### ISSUE TITLE
Union false positive introduced by 1.1.182

### ISSUE THREAD

**Issue by rsokl** [microsoft/pyright#2499]
**Describe the bug**
Source code involving a bare `Union` as an object -- not an annotation -- is being flagged by pyright-basic.

**To Reproduce**
```python
from typing import get_origin, Union

x = Union[int, str]

origin = get_origin(x)
assert origin is Union  # error: Union requires two or more type arguments
```

**Expected behavior**
It seems like this usage of `Union` is valid.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.183, which I just published. It will also be included in the next release of pylance.
